api/service.py

```
@@ -590,6 +649,93 @@ async def get_anomaly_history(
+    return UserResponse(
+        id=current_user.id,
+        username=current_user.username,
+        role=current_user.role.value,
+        email=current_user.email,
+        created_at=current_user.created_at,
+        is_active=current_user.is_active
+    )
+
+
+@app.post("/api/v1/auth/apikeys", response_model=APIKeyCreateResponse)
+async def create_api_key(request: APIKeyCreateRequest, current_user: User = Depends(get_current_user)):
```

Ruff (0.14.10)

Undefined name `User`

(F821)